app/src/main/java/com/example/tasky/authentication/presentation/components/ButtonWithLoader.kt

```diff
@@ -10,15 +10,14 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
-import com.example.tasky.R
 import com.example.tasky.ui.theme.Black
 
 @Composable
 fun ButtonWithLoader(
+    text: String,
```

Comment:
Modifier should be after the "isLoading"
Or else, the user needs to access a mandatory parameter (isLoading) through named parameters